core/src/util/usFramework.cpp

```diff
@@ -72,7 +73,7 @@ void Framework::Start()
       d->operation = BundlePrivate::OP_ACTIVATING;
       break;
     case STATE_ACTIVE:
-      return;
+      break;
```

Comment:
Ok. I was looking at Felix as a reference for this.
Section 4.2.5 does help but I still think that the statement
> After the system bundle enters the ACTIVE state, a Framework STARTED event is broadcast.

is ambiguous. It doesn't address concurrency. Section 10.2.2.11 doesn't offer much help either. 

I'll revert back to the ```return```. There is no clear need or use case to send Framework STARTED events multiple times.

I wasn't referring to any passage in particular. I was referring to the absence of any statement to the effect of "only send the Framework STARTED event once" in the OSGi spec.